packages/common/src/ui/layout/tables/columns/NameAndColorColumn.tsx

```diff
@@ -1,20 +1,22 @@
 import React, { useState } from 'react';
-import { CellProps, Column } from 'react-table';
 import { Color, ColorMenu, UnstyledIconButton } from '../../../components';
 import { Circle } from '../../../icons';
 import { Box } from '@mui/system';
+import { DomainObject } from '../../../..';
+import { ColumnDefinition } from '../columns/types';
 
-type Colorable = {
+interface DomainObjectWithRequiredFields extends DomainObject {
   color: string;
-} & Record<string, unknown>;
+  name: string;
+}
```

Comment:
this is cute, it means I can't use that column for the column set in the `GeneralTab`